dtm0/it/all2all/README.md

```diff
@@ -92,3 +92,21 @@ Now reload the daemon & stop the hare-consul-agent using following command
 [root@ssc-vm-c-1966 hare]# sudo systemctl daemon-reload && sleep 5 && service hare-consul-agent stop 
 ```
 Try running the test again. Refer [this](https://github.com/Seagate/cortx-motr/blob/dtm0-main/dtm0/it/all2all/README.md#command-to-run-this-test)
+
+## Known Issues
+### 1. all2all integration test performs all tests and at the end shows test status as FAIL because of m0d service failed.
+Example test log is as shown below:
+```sh
+ERR: Process m0d@0x7200000000000001:0xc.service failed
+ERR: Process m0d@0x7200000000000001:0x1a.service failed
+ERR: Process m0d@0x7200000000000001:0x28.service failed
+ERR: TEST STATUS: FAIL
+```
+In above case, in journalctl logs if we see the m0d services getting killed because of timed out like shown below, then it is not an issue with all2all test.
+```sh
+m0d@0x7200000000000001:0x28.service stop-sigterm timed out. Killing
```

Comment:
What is the real root cause of this killing which leads to dtm0 test failure?

If we got this issue, what is the solution?
Test it again?

Reply:
@ivan-alekhin  could you please comment?

Reply:
> What is the real root cause of this killing which leads to dtm0 test failure?

The root cause may be related to https://github.com/Seagate/cortx-hare/pull/1606. Once the upstream gets a solution for this timeout issue, we will verify it to ensure it has fixed the all2all test too. @rajatpatil98, could you please add a link to the ticket (https://jts.seagate.com/browse/EOS-19683) here?

> If we got this issue, what is the solution? Test it again?

Yes -- either run the test again or look into the log to confirm that the process was killed by systemd due to the timeout. That does not sound good but that all we have at the moment.

Reply:
updated issue and added link to the ticket EOS-19683.